### ISSUE TITLE
Can not create the property title on NPC before it exists.

### ISSUE THREAD

**Issue by bryanjnelson** [edgedb/easy-edgedb#113]
In the section **Using the ‘for’ and ‘union’ keywords** in **Chapter 9** the following statement is mentioned:

`update NPC filter .name = 'John Seward'
set { title := 'Dr.' };
`

Perhaps `NPC.title` existed at some point in another iteration of this documentation, but at the current time it does not exist by Chapter 9.

Potential solutions:

1. Remove this statement
2. Add NPC.title to the schema in a previous chapter
3. Move this statement to a later point in the book where NPC.title does exist?

**Comment by Dhghomon** [comment]
Yes, I noticed this last week during my ongoing review on this branch where it is fixed: https://github.com/edgedb/easy-edgedb/tree/self-review Hopefully will reach the end of it in a few days and able to merge it.

**Comment by Dhghomon** [comment]
The branch has been merged so it's been solved with solution #3 you mentioned, thanks!
